curriculum/challenges/english/25-front-end-development/quiz-javascript-comparisons-and-conditionals/66edc47c11492ac5cf258ad9.md

```diff
@@ -17,439 +17,501 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder distractor 3
+The statement will throw an error.
 
 #### --answer--
 
-Placeholder answer
+The statement will run.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+if (2 == "2") {
+  const a = 2;
+}
+console.log(a);
+```
+
+What does it output?
 
 #### --distractors--
 
-Placeholder distractor 1
+2
 
 ---
 
-Placeholder distractor 2
+Nothing
 
 ---
 
-Placeholder distractor 3
+`undefined`
 
 #### --answer--
 
-Placeholder answer
+Error
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens when you don't include `break` while implementing a switch statement?
 
 #### --distractors--
 
-Placeholder distractor 1
+The switch statement will stop abruptly.
 
 ---
 
-Placeholder distractor 2
+The switch statement will throw an error after previous statement.
 
 ---
 
-Placeholder distractor 3
+The switch statement will exit after the first match.
 
 #### --answer--
 
-Placeholder answer
+The code continues to execute the next case statement, regardless of whether the case matches.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+const a = 2;
+if (1 == "1") {
+  let b = 3;
+  console.log(a + b);
+}
+console.log(b);
+```
+
+What will be the output of this code?
+
+
 
 #### --distractors--
 
-Placeholder distractor 1
+Error.
 
 ---
 
-Placeholder distractor 2
+5, followed by 3
 
 ---
 
-Placeholder distractor 3
+5 followed by undefined
 
 #### --answer--
 
-Placeholder answer
+5 followed by Error
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What keyword used in a switch statement to handle cases when all of the specified cases are false?
 
 #### --distractors--
 
-Placeholder distractor 1
+`then`
 
 ---
 
-Placeholder distractor 2
+`after`
 
 ---
 
-Placeholder distractor 3
+`else`
 
 #### --answer--
 
-Placeholder answer
+`default`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does this operator `&&` represent?
 
 #### --distractors--
 
-Placeholder distractor 1
+XOR
 
 ---
 
-Placeholder distractor 2
+OR
 
 ---
 
-Placeholder distractor 3
+NOT
 
 #### --answer--
 
-Placeholder answer
+AND
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+let x = 5;
+if (x > 1 && x < 10) {
+  console.log("x is between 1 and 10");
+} else {
+  console.log("x is not between 1 and 10");
+}
+```
+
+What does the code evaluates to?
 
 #### --distractors--
 
-Placeholder distractor 1
+Error.
 
 ---
 
-Placeholder distractor 2
+No output.
 
 ---
 
-Placeholder distractor 3
+x is not between 1 and 10
 
 #### --answer--
 
-Placeholder answer
+x is between 1 and 10
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the operator `!` do?
 
 #### --distractors--
 
-Placeholder distractor 1
+It will reverse the value of a variable.
 
 ---
 
-Placeholder distractor 2
+It will do "AND" operation.
 
 ---
 
-Placeholder distractor 3
+It will do "OR" operation.
 
 #### --answer--
 
-Placeholder answer
+It will negate the boolean expression.
```

Comment:
It will also negate non-Booleans, ie.:
```js
console.log(!3)  // false
console.log(!'')  // true
```

I'm not sure right now how to nicely put it in wording.